devsetup/scripts/ipv6-nat64/sno.sh

```diff
@@ -75,7 +75,7 @@ ARCH="${ARCH:-x86_64}"
 MEMORY="${MEMORY:-32768}"
 VCPUS="${VCPUS:-12}"
 OS_VARIANT="${OS_VARIANT:-fedora-coreos-stable}"
-DISK_SIZE="${DISK_SIZE:-150}"
+DISK_SIZE="${DISK_SIZE:-31}"
```

Comment:
Is this possibly going to cause issues for some CI jobs since we use local-storage? Or do we think 31GB should be sufficient for all of our jobs?

Reply:
I just want to align this with CRC. Jobs will be overriding this value for their requirements where necessary